compendium/DeclarativeServices/src/manager/ConfigurationNotifier.cpp

```diff
@@ -113,7 +115,7 @@ namespace cppmicroservices
             }
         }
         bool
-        ConfigurationNotifier::AnyListenersForPid(std::string const& pid) noexcept
+        ConfigurationNotifier::AnyListenersForPid(std::string const& pid, std::shared_ptr<cppmicroservices::AnyMap> properties) noexcept
```

Comment:
`properties` should be passed into this function as a `const ref AnyMap`. I don't see a need to wrap it in a shared_ptr.